bool ActivateLeftPaletteDetection
bool ActivateRightPaletteDetection
bool ActivateAllZones